integrationTests/p2p/peerDiscovery/testRunnner.go

```diff
@@ -16,7 +16,7 @@ import (
 	libp2pCrypto "github.com/libp2p/go-libp2p-core/crypto"
 )
 
-var durationMsgRecieved = time.Duration(time.Second * 2)
+var durationMsgRecieved = 2 * time.Second
```

Comment:
Add comments for exported functions in this file.

Reply:
Also, you could remove CreateMessenger(), and modify and use CreateMessengerWithKadDht() from test initializer.

Reply:
done